darling_park_cs:
  transform: [-0.29794830207371, 0.3141416639957557, 0.6104152402681149, -1.544164822838139, 0.03312939577570084, -0.01121896959815881]
  parent_frame: ar_marker_4
darling_unpark_cs:
  transform: [-0.2865144126248639, 0.3261686638737945, 0.6666893993499646, -1.537451444448914, 0.05447879700557728, -0.01135561986516054]
  parent_frame: ar_marker_4
